tests/basic_client.py

```
@@ -0,0 +1,62 @@
+    """
+    while True:
+        try:
+            message = client_socket.recv(1024).decode('utf-8')
+            if not message:
+                print("Connection closed by the server.")
+                break
+            print(f"Received: {message}")
+        except ConnectionResetError:
+            print("Connection was forcibly closed by the remote host.")
+            break
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)